spotinst/resource_spotinst_organization_user_test.go

```diff
@@ -119,29 +119,12 @@ func TestAccSpotinstOrganization_User(t *testing.T) {
 					resource.TestCheckResourceAttr(userResourceName, "policies.1.policy_account_ids.0", "act-e2be553a"),
 				),
 			},
-			{
-				Config:             createOrganizationUserTerraform(testOrganization_User_Update_Policy, userName, userPassword),
```

Comment:
Why this test asserts are deleted.

Reply:
This part is duplicate which is not required.